src/main/java/org/gradle/profiler/mutations/ApplyNonAbiChangeToJavaSourceFileMutator.java

```diff
@@ -15,9 +16,10 @@ public ApplyNonAbiChangeToJavaSourceFileMutator(File sourceFile) {
     }
 
     @Override
-    protected void applyChangeTo(CompilationUnit compilationUnit) {
+    protected void applyChangeTo(BuildContext context, CompilationUnit compilationUnit) {
         MethodDeclaration existingMethod = getExistingMethod(compilationUnit);
-        existingMethod.getBody().get().addStatement(0, JavaParser.parseStatement("System.out.println(\"" + getUniqueText() + "\");"));
+        existingMethod.getBody()
+            .ifPresent(body -> body.addStatement(0, JavaParser.parseStatement("System.out.println(\"" + context.getUniqueBuildId() + "\");")));
```

Comment:
I think it is better to fail here instead of doing no change to the body of the method. This way at least you know that the non-ABI change was not successfully applied.

Reply:
Good point.